segments_customerio.go

```diff
@@ -22,41 +22,46 @@ const (
 )
 
 // AddPeopleToSegment adds people to a segment.
-func (c *CustomerIO) AddPeopleToSegment(ctx context.Context, segmentID int, idType string, ids []string) error {
+func (c *CustomerIO) AddPeopleToSegment(ctx context.Context, segmentID int, ids []string) error {
 	if segmentID == 0 {
 		return ParamError{Param: "segmentID"}
 	}
 	if len(ids) == 0 {
 		return ParamError{Param: "ids"}
 	}
 	return c.request(ctx, "POST",
-		fmt.Sprintf("%s/api/v1/segments/%d/add_customers?id_type=%s", c.URL, segmentID, c.getValidIDType(idType)),
+		fmt.Sprintf("%s/api/v1/segments/%d/add_customers%s", c.URL, segmentID, c.getQueryParams()),
```

Comment:
what about using the url package to make the url instead? I think it's methods can help build the url in a cleaner way

Reply:
I agree with you, but since most of the code blocks are structured that way, I preferred not to modify how the URLs are prepared.
For instance here: https://github.com/captiv8io/go-customerio/blob/main/customerio.go#L99

Reply:
Will approve since it is a hotfix, but I feel we should improve this code